services/brig/src/Brig/User/EJPD.hs

```diff
@@ -60,7 +67,7 @@ ejpdRequest includeContacts (EJPDRequestBody handles) = do
       let uid = userId target
 
       ptoks <-
-        PushTok.tokenText . view PushTok.token <$$> wrapHttp (Intra.lookupPushToken uid)
+        PushTok.tokenText . view PushTok.token <$$> liftSem (NotificationSubsystem.getPushTokens uid)
```

Comment:
I personally find this a bit long. Should we agree on a short form like we do for effects with `E`? Maybe just `Sub`, `Notifs`, `Notifications`, etc. ...?

Reply:
`Notifs` and `Notifications` would collide with those types, `E` and `Sub` doesn't convey much. I will try to get rid of these qualified imports and use them as a last resort.